.gitlab-ci.yml

```
@@ -917,7 +890,7 @@ agent_rpm-x64-a7:
 agent_rpm-arm-a6:
   stage: package_build
   needs: ["run_go_tidy_check", "build_system-probe-arm64"]
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/rpm_arm64:$DATADOG_AGENT_ARMBUILDIMAGES
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/rpm_arm64:v2424505-0439a40
```

I believe this should be
```suggestion
  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/rpm_arm64:$DATADOG_AGENT_BUILDIMAGES
```

---

That is what I thought at first but the current tag is the one for normal images, not arm images
https://github.com/DataDog/datadog-agent/blob/99379f2e34a66af036a6f2bac582fb5da0d27092/.gitlab-ci.yml#L85-L89

---

I think the reason behind that is that he wanted to test only the new 32-bit arm images first without changing the 64-bit arm images because they were failing (you can see that before this PR, `$DATADOG_AGENT_ARMBUILDIMAGES` was `v2424505-0439a40` too).

Now that the current `$DATADOG_AGENT_ARMBUILDIMAGES` image succeeds on both 32-bit and 64-bit arm, I don't see a reason why we would not use it there.

---

👍, marking these as solved to avoid repetition